Fw/Log/LogString.cpp

```diff
@@ -28,7 +28,7 @@ namespace Fw {
     }
 
     NATIVE_UINT_TYPE LogStringArg::length() const {
-        return (NATIVE_UINT_TYPE) strnlen(this->m_buf,sizeof(m_buf));
+        return strnlen(this->m_buf,sizeof(m_buf));
```

Comment:
Here we have an implicit cast from size_t to NATIVE_UINT_TYPE which on some systems can produce warnings about loss of precision.